### PR TITLE
Fix test case

### PR DESCRIPTION
**Pull request by bennahugo**
- [ X] Tests added / passed
- [ X] Fully documented, including `HISTORY.rst` for all changes
      and one of the `docs/*-api.rst` files for new API

Note: Test cases to pass on CI server as a check before merging,
modified Jenkins file. Don't merge until pass

<details>
<summary> Howto run test cases and lint the code base </summary>

```bash
$ py.test --flake8 -v -s tricolour
```

If you encounter flake8 failures, a quick way to correct
this is to run `autopep8` and `flake8` again.

```bash
$ pip install -U autopep8
$ autopep8 -r -i tricolour
$ flake8 tricolour
```

</details>

<details>
<summary> Howto build the documentation </summary>

To build the docs locally:

```bash
$ pip install -r requirements.readthedocs.txt
$ cd docs
$ READTHEDOCS=True make html
```

</details>

### CODE REVIEW

File: tricolour/tests/test_acceptance.py [hunk 1]
```diff
@@ -14,43 +14,16 @@
...
-
-    response = session.get(URL, params={'id': id}, stream=True)
-    token = _get_confirm_token(response)
-
-    if token:
-        params = {'id': id, 'confirm': token}
-        response = session.get(URL, params=params, stream=True)
-
-    _save_response_content(response, destination)
+    URL = "https://drive.google.com/uc?id={}".format(_GOOGLE_FILE_ID)
```
Comment (sjperkins) [inline tricolour/tests/test_acceptance.py:25]:
```suggestion
    URL = f"https://drive.google.com/uc?id={_GOOGLE_FILE_ID}"
```

File: tricolour/tests/test_acceptance.py [hunk 2]
```diff
@@ -14,43 +14,16 @@
 
 from pyrap.tables import table as tbl
 import numpy as np
-import requests
+import gdown
```
Comment (sjperkins) [inline tricolour/tests/test_acceptance.py:17]: nice, I didn't know about this.

File: setup.py [hunk 3]
```diff
@@ -46,6 +46,6 @@
     packages=find_packages(),
     python_requires=">=3.5",
     url='https://github.com/ska-sa/tricolour',
-    version='0.1.8',
+    version='0.1.8.1',
```
Comment (sjperkins) [inline setup.py:49]:
```suggestion
    version='0.1.9',
```

File: HISTORY.rst [hunk 4]
```diff
@@ -2,6 +2,12 @@
 History
 =======
 
+0.1.8.1 (2022-09-29)
```
Comment (sjperkins) [inline HISTORY.rst:5]: Version numbers are major.minor.patch. I think we can just bump the patch version here.

```suggestion
0.1.9 (2022-09-29)
```

File: setup.py [hunk 5]
```diff
@@ -11,15 +11,15 @@
...
+    'numpy >= 1.14.0, <= 1.19.5', # breakage in newer numpy + numerical errors
     'numba >= 0.43.0',
     'scipy >= 1.2.0',
     'threadpoolctl >= 1.0.0',
     'dask-ms == 0.2.6',
     'zarr >= 2.3.1'
 ]
 
-extras_require = {'testing': ['pytest', 'pytest-flake8', 'requests']}
+extras_require = {'testing': ['pytest', 'pytest-flake8', 'flake8 >= 4.0.0, <5.0.0', 'requests', 'gdown']}
```
Comment (sjperkins) [inline setup.py:22]: Pinning flake8 < 5 is due to https://github.com/tholo/pytest-flake8/issues/87#issuecomment-1201760236.

pre-commit hooks may be the way forward here, in future PRs

File: setup.py [hunk 6]
```diff
@@ -46,6 +46,6 @@
     packages=find_packages(),
     python_requires=">=3.5",
```
Comment (sjperkins) [inline setup.py:47]: Could we bump the python minor version up to 3.8?

```suggestion
    python_requires=">=3.8",
```

**Review by sjperkins — CHANGES_REQUESTED** [review]
These changes look good. I think the only change I feel should be made is to change the release version from 0.1.8.1 to 0.1.9.  Feel free to defer other suggestions to future PRs

File: setup.py (hunk 5 above)
Comment (bennahugo) [reply setup.py:22]: Yea I'm in a bit of a fix all jenkins spree and I would like to get away with minimal changes in this PR. I will leave that till the next round where I can hopefully get to document and integrate the stuff @JSKenyon has been working on

File: setup.py (hunk 6 above)
Comment (bennahugo) [reply setup.py:47]: I will meet you midway and upgrade to 3.6, and supply a 3.8 build as well (like I do for DDFacet and KillMS)

File: HISTORY.rst (hunk 4 above)
Comment (bennahugo) [reply HISTORY.rst:5]: noted and done

File: setup.py (hunk 3 above)
Comment (bennahugo) [reply setup.py:49]: noted and done

File: tricolour/tests/test_acceptance.py (hunk 2 above)
Comment (bennahugo) [reply tricolour/tests/test_acceptance.py:17]: Yup damned google innovation has broken basic requests compatibility in their API...

File: tricolour/tests/test_acceptance.py (hunk 1 above)
Comment (bennahugo) [reply tricolour/tests/test_acceptance.py]: ok moved to 3.6+ so switched to formatstrings

**Comment by bennahugo** [comment]
Ok things are back to green @sjperkins perhaps you can just tick. I have addressed by moving to >=3.6 and including a test for 3.8

**Review by sjperkins — APPROVED** [review]
approved!

**Pull request merged**